scripts/bdairo.js

```diff
@@ -1,3 +1,10 @@
 let name= "Boluwarin Dairo";
-let id = "INTERNSHIP ID: HNG-01402"
-console.log(" Hello World, This is " + name + " with " + id +" using javascript for stage 2 task" );
```

Comment:
You are missing your email from your output string. Also, there are some grammatical errors in your output (e.g. there is an unnecessary space at the beginning of your output string).

See [the goal](https://github.com/hng-teamfury/team-fury-1/blob/dev/README.md#the-goal) section of the README file for an example of how the output string should be formatted.